python/add_corona.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+#===================================================
+#
+#    Copyright (c) 2025
+#      SMASH Hybrid Team
+#
+#    GNU General Public License (GPLv3 or later)
+#
+#===================================================
+
+import os
+import sys
+import argparse
+import numpy as np
+
+def read_corona_file(file_path):
+    with open(file_path, 'r') as f:
+        lines = [line.strip().split() for line in f if not line.startswith('#')]
+    array = np.atleast_2d(np.array(lines, dtype=str)) if len(lines) != 0 else []
+    return array
+
+def initialize_final_file(file_path):
+    header_string = "#!OSCAR2013 particle_lists t x y z mass p0 px py pz pdg ID charge \n\
+# Units: fm fm fm fm GeV GeV GeV GeV GeV none none e \n\
+# SMASH-3.3 \n"
+    with open(file_path, 'w') as f:
+        f.write(header_string)
+
+def read_Sampler_file(file_path, file_path_new, array_to_append):
+    # Open the file and read lines
+    lines_buffer = np.empty((0, 12), dtype=str)
+    event_no = 0
+    with open(file_path, 'r') as file:
+        for line in file:
+            if '#!OSCAR2013' in line or 'Units:' in line or 'SMASH-' in line:
+                continue
+            if 'end' in line:
```

Comment:
Is there a reason why you didn't choose (the safer) `elif` here?
```suggestion
            elif 'end' in line:
```